contracts/contracts/coordination/Coordinator.sol

```diff
@@ -104,6 +104,7 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
     IReimbursementPool internal reimbursementPool;
     mapping(address => ParticipantKey[]) internal participantKeysHistory;
     mapping(bytes32 => uint32) internal ritualPublicKeyRegistry;
+    Participant internal sentinelParticipant;
```

Comment:
In order to avoid potential problems later with storage layout if `Participant` struct changes and/or more state variables are added to the contract, we can change the current sentinel for the following:
```suggestion
    mapping(uint256 => Participant) internal sentinelParticipants;
    uint256 internal sentinelIndex = 0;
```
Later on, we just need to assign `sentinelParticipant = sentinelParticipants[sentinelIndex] when needed`.

Reply:
Updated - let us know if we did it correct (cc @vzotova ).

Reply:
I do not see anything wrong, but also I don't get everything about those variables

Reply:
@cygnusv 's initial concern was around the usage of a `sentinelParticipant` slot for a Participant struct which could change size/structure later on down the road.

By using a mapping we don't have to worry about the struct/slot potentially changing size.

(@cygnusv hopefully I explained this properly 😅 )

Reply:
I got the idea, in that case do we need index field for that? if value always will be only one then it's not necessary
other way to do that, a bit more straightforward -> put some gap between regular `sentinelParticipant` struct and previous values, like `uint256[20]`, in that case extending will be same easy, and gap is not taking any gas

Reply:
We (@KPrasch and myself) defer to you and @cygnusv for your expertise on the best course of action here ... happy to make adjustments as needed.